src/main/java/emu/grasscutter/BuildConfig.java

```diff
@@ -0,0 +1,7 @@
+
+        package emu.grasscutter;
+        public class BuildConfig {
```

Comment:
Remove.

Reply:
toxic

Reply:
at least ask nicely like "Hi Lunatic please remove this line 🥺 uwu"

Reply:
> Remove.

actually you have to remove it because you are the @Write access 💀